simulator/Makefile

```diff
@@ -5,7 +5,7 @@ generate:
 
 .PHONY: lint
 lint:
-	golangci-lint run --timeout 30m ./...
+	golangci-lint run --timeout 90m ./...
```

Comment:
It's weird that we had 30min timeout, but nonetheless, the job had been running more than 30 min.
Here, I increased the timeout, plus let's increase the resource for the job a bit: https://github.com/kubernetes/test-infra/pull/31626